microovn/metadata.yaml

```diff
@@ -10,7 +10,7 @@ description: |
 maintainers:
   - OVN Engineering <ubuntu-ovn-eng@lists.launchpad.net>
 
-base: ubuntu@24.04
+base: ubuntu@25.03
```

Comment:
See above comment, in a multi platform world we likely can no longer populate this key in `metadata.yaml`.

Maybe charmcraft dtrt wrt. popualting this file for us now?